package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "CsvParser",
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {}
+}
```

Comment:
I think this file must have been generated by accident as we don't have a package.json / node in this repo.